pkg/cri/server/podsandbox/sandbox_stop.go

```diff
@@ -22,52 +22,56 @@ import (
 	"syscall"
 	"time"
 
+	"github.com/containerd/log"
+
 	eventtypes "github.com/containerd/containerd/v2/api/events"
 	"github.com/containerd/containerd/v2/errdefs"
+	"github.com/containerd/containerd/v2/pkg/cri/server/podsandbox/types"
 	sandboxstore "github.com/containerd/containerd/v2/pkg/cri/store/sandbox"
 	ctrdutil "github.com/containerd/containerd/v2/pkg/cri/util"
 	"github.com/containerd/containerd/v2/protobuf"
 	"github.com/containerd/containerd/v2/sandbox"
-	"github.com/containerd/log"
 )
 
 func (c *Controller) Stop(ctx context.Context, sandboxID string, _ ...sandbox.StopOpt) error {
-	sandbox, err := c.sandboxStore.Get(sandboxID)
-	if err != nil {
-		return fmt.Errorf("an error occurred when try to find sandbox %q: %w",
-			sandboxID, err)
+	podSandbox := c.store.Get(sandboxID)
+	if podSandbox == nil {
+		return nil
```

Comment:
Could you consider return `ErrNotFound` here?

Reply:
done, and let cri server log and ignore the ErrNotFound.